oap-server/server-starter/src/main/resources/ui-initialized-templates/mysql/mysql-service.json

```diff
@@ -0,0 +1,351 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+[
+   {
+      "id":"2981ef0e-1552-4cf0-a68c-da023b4a0fd8",
+      "configuration":{
+         "children":[
+            {
+               "x":0,
+               "y":0,
+               "w":6,
+               "h":11,
+               "i":"0",
+               "type":"Widget",
+               "metricTypes":[
+                  "readMetricsValue"
+               ],
+               "metrics":[
+                  "meter_mysql_uptime"
+               ],
+               "graph":{
+                  "type":"Card",
+                  "fontSize":40,
+                  "textAlign":"center",
+                  "showUnit":true
+               },
+               "widget":{
+                  "title":"MySQL Uptime"
```

Comment:
suggest adding a unit in the title `MySQL Uptime (s)` or MySQL Uptime (min) ?

Reply:
Is the uptime a timestamp? Or a duration from booting?

Reply:
It is a duration from booting. Its unit should be day. I forget to add it. I will do it right now.